source/upgrade.txt

```diff
@@ -403,46 +411,49 @@ Version 4.0 Breaking Changes
 
 This driver version introduces the following breaking changes:
 
-- Several classes and methods marked as deprecated in the 3.12 release
-  were removed in this version.
-- The insert helper methods return an insert result object instead of
+- Removes several classes and methods marked as deprecated in the 3.12 release.
+- Modifies insert helper methods to return an insert result object instead of
```

Comment:
S: Might be easier to parse if you used the actual class name: "to return an ``InsertResult`` object" (or whatever it's called)